tests/PSRule.Rules.Azure.Tests/Azure.ContainerApp.Tests.ps1

```diff
@@ -52,5 +52,22 @@ Describe 'Azure.ContainerApp' -Tag 'ContainerApp' {
             $ruleResult.Length | Should -Be 1;
             $ruleResult.TargetName | Should -BeIn 'capp-A';
         }
```

Comment:
You may need to bump `Azure.ContainerApp.Insecure` for the new resource instances.